src/lib/configs/initConfig.ts

```diff
@@ -456,7 +456,7 @@ export function getReadonlyConfigInitFunction<
   };
 }
 
-const initializedConfigs = new Set<string>();
```

Comment:
Why not just `configs` ?

Reply:
it probably can be just `configs` but I suggest to leave it as it is, it's a map of the configs that were initialized previously, so here seems reasonable to mention this fact in the variable name